compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveCustomOpAddPass.h"
+
+#include "flatbuffers/flexbuffers.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/AttrFusedActFunc.h>
+
+namespace
+{
+
+/** BEFORE
+ *                                  [CircleConst]
+ *                                        |
+ *        [CircleNode]         [BroadcastTo(CircleCustom)]
+ *              \                         |
+ *               \                [CircleCustomOUt]
+ *                \                   /
+ *               [AddV2(CircleCustom)]
+ *  AFTER
+ *
+ *         [CircleConst]         [CircleNode]
+ *                   \           /
+ *                    \         /
+ *                    [CircleAdd]
+ */
+bool resolve_with_BroadcastTo(luci::CircleCustom *addv2)
+{
+  const luci::CircleCustomOut *input1 =
+      dynamic_cast<const luci::CircleCustomOut *>(addv2->inputs(1));
+
+  if (input1 != nullptr)
```

Comment:
what if `addv2->inputs(0)` is `BroadcastTo` ?

Reply:
@seanshpark 
Well, I followed our in-house model because already this pass is highly dependent on our model. If some model has `AddV2` which has `BroadcastTo` as first argument, I'll implement it then.

Reply:
How about introduce a function that returns 0 or 1 or -1 as idx
- 0 : `BroadcastTo` is input(0)
- 1: `BroadcastTo` is input(1)
- -1: there is no `BroadcastTo` input
and then we can use this value for `BroadcastTo` and `1 - idx` as opposite.